build/meta.mk

```diff
@@ -1,6 +1,5 @@
 # Variables
 BUILD_SOURCES := \
-	$(wildcard build/**/*) \
```

Comment:
I believe it's not since I removed the two folders in there, `css-minify` and `css-merge`, I think it's not necessary anymore. I'm not entirely sure about this, though, so let me know otherwise!

Reply:
Ah right it was for the build scripts. But this line would still be necessary because it checks for the `*.mk` files as well. So this should be re-added but changed to `$(wildcard build/*)`